packages/adapter-mysql/src/index.ts

```diff
@@ -0,0 +1,273 @@
+import type {
+  Adapter,
+  AdapterUser,
+  VerificationToken,
+  AdapterSession,
+} from "@auth/core/adapters";
+import type { OkPacketParams, Pool } from "mysql2/promise";
+// type k = AdapterUser & { password?: string };
+export const mapExpiresAt = (account: any): any => {
+  const expires_at: number = parseInt(account.expires_at);
+  return {
+    ...account,
+    expires_at,
+  };
+};
+
+const mysqlAdapter = (client: Pool): Adapter => {
+  return {
+    async createVerificationToken(
+      verificationToken: VerificationToken
+    ): Promise<VerificationToken> {
+      const { identifier, expires, token } = verificationToken;
+      const sql = `
+        INSERT INTO verification_token ( identifier, expires, token ) 
+        VALUES (?, ?, ?)
+        `;
+      await client.query(sql, [identifier, expires, token]);
+      return verificationToken;
+    },
+    async useVerificationToken({
+      identifier,
+      token,
+    }: {
+      identifier: string;
+      token: string;
+    }): Promise<VerificationToken | null> {
+      const sql = `select* from verification_token
```

Comment:
You have syntax issue 
```suggestion
      const sql = `select * from verification_token
```